src/scripts/browserDeviceProfile.js

```diff
@@ -244,12 +244,14 @@ function supportedDolbyVisionProfilesHevc(videoTestElement) {
     if (videoTestElement.canPlayType) {
         if (videoTestElement
             .canPlayType('video/mp4; codecs="dvh1.05.09"')
-            .replace(/no/, '')) {
+            .replace(/no/, '')
+            || browser.xboxOne) {
```

Comment:
I would move this to the beginning of the function and just do

```js
if (browser.xboxOne) return [ 5, 8 ];
```

Reply:
Thanks done.